packages/app-security/src/components/NotAuthorizedError.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import { Link } from "@webiny/react-router";
+import { css } from "emotion";
+import styled from "@emotion/styled";
+import Helmet from "react-helmet";
+import authErrorImg from "../admin/assets/images/SecureRouteError.svg";
+import { Typography } from "@webiny/ui/Typography";
+
+const ContentWrapper = styled("div")({
+    display: "block",
+    paddingTop: "15%",
+    textAlign: "center",
+    margin: "auto"
+});
+
+const styles = {
+    authErrorImgStyle: css({
+        width: "192px"
+    }),
+    bodyStyle: css({
+        color: "var(--mdc-theme-text-primary-on-background)"
+    }),
+    linkStyle: css({
+        textDecoration: "none",
+        "&:hover": {
+            textDecoration: "none"
+        }
+    }),
+    divStyles: css({
+        padding: "5px"
```

Comment:
Please don't add bottom spacing between each line of text. It's a single paragraph of text should look like that.

Reply:
Okay will remove, I added it coz it looked good